### PR TITLE
feat: http receiver structure and misc. improvements

### PR DESCRIPTION
**Pull request by anthonypillot**
Related to #10 ; still in progress. Here is just the structure.

### CODE REVIEW

File: src/interfaces/service.interface.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export interface ServiceInterface {
+    getGames(): Promise<string>
+    getPrices(): Promise<string>;
```
Comment (Bedhiver) [inline src/interfaces/service.interface.ts:3]: We don't need prices. I don't get this information from Epic Games

File: src/services/epicGames.service.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import client from "../outputs/epic-games/client";
+
+import { ServiceInterface } from "../interfaces/service.interface";
+
+export class EpicGamesService implements ServiceInterface {
+    public async getGames(): Promise<string> {
+        const data = await client.getDatas();
+        return "games";
+    }
+    public async getPrices(): Promise<string> {
```
Comment (Bedhiver) [inline src/services/epicGames.service.ts:10]: Same thing as `src/interfaces/service.interface.ts`

**Review by Bedhiver — CHANGES_REQUESTED** [review]

File: src/interfaces/service.interface.ts (hunk 1 above)
Comment (anthonypillot) [reply src/interfaces/service.interface.ts]: Fixed by 7a8ef43b52df382ca8af5451b7e23e31421f126b

File: src/services/epicGames.service.ts (hunk 2 above)
Comment (anthonypillot) [reply src/services/epicGames.service.ts]: Fixed by 7a8ef43b52df382ca8af5451b7e23e31421f126b

**Review by Bedhiver — APPROVED** [review]

**Review by Bedhiver — APPROVED** [review]

**Pull request merged**